server/portal/apps/accounts/urls.py

```diff
@@ -3,25 +3,18 @@
    :synopsis: Accounts URLs
 """
 from django.conf.urls import url
-from portal.apps.accounts.views import LogoutView
+from portal.apps.accounts.views import LogoutView, accounts
 from portal.apps.accounts import views
-from portal.apps.workbench.views import IndexView as ReactView
 
 
 app_name = 'portal_accounts'
 urlpatterns = [
     url(r'^logout/?', LogoutView.as_view(), name='logout'),
-
-    # Endpoints for React based profile-page
-    url(r'^profile/', ReactView.as_view(), name='manage_profile'),
+    url(r'^profile/', accounts, name='manage_profile'),
```

Comment:
This is good for backward compatibility, but can we change the url in the dropdown to `workbench/account` as well?

Reply:
Done.